diff -U0 -U 3 -Nrp lektor.orig/admin/static/app.js lektor/admin/static/app.js
--- lektor.orig/admin/static/app.js	2023-06-18 15:24:57.275254529 +0300
+++ lektor/admin/static/app.js	2023-06-18 15:26:30.699362535 +0300
@@ -2229,13 +2229,13 @@ var charmap = {
   Ó: "O",
   Ô: "O",
   Õ: "O",
-  Ö: "Oe",
+  Ö: "O",
   Ő: "O",
   Ø: "O",
   Ù: "U",
   Ú: "U",
   Û: "U",
-  Ü: "Ue",
+  Ü: "U",
   Ű: "U",
   Ý: "Y",
   Þ: "TH",
@@ -5872,4 +5872,4 @@ module.exports = JSON.parse('{"RETURN_TO
 /******/ 	
 /******/ })()
 ;
-//# sourceMappingURL=app.js.map
\ No newline at end of file
+//# sourceMappingURL=app.js.map
diff -U0 -U 3 -Nrp lektor.orig/publisher.py lektor/publisher.py
--- lektor.orig/publisher.py	2023-06-18 15:25:11.879270966 +0300
+++ lektor/publisher.py	2023-06-18 15:25:25.779286762 +0300
@@ -93,6 +93,7 @@ def _temporary_folder(env):
     finally:
         try:
             shutil.rmtree(folder)
+            os.rmdir(base)
         except (IOError, OSError):
             pass
 
